src/components/tag/_tag.scss

```diff
@@ -66,3 +66,90 @@
     }
   }
 }
+
+@mixin tags--x {
+  .#{$prefix}--tag {
+    @include font-family;
+    @include typescale('omega');
+    display: inline-flex;
+    align-items: center;
+    padding: 0 $spacing-xs;
+    height: 1.5rem;
+    margin: $spacing-2xs;
+    border-radius: rem(15px);
+
+    &:not(:first-child) {
+      margin-left: 0;
+    }
+
```

Comment:
Do we need a `hover`? If these are clickable, we should have a `cursor: pointer`. If not, we should have `cursor: default`. Currently, I'm getting an input icon on hover?

Reply:
No there should not be a hover.